llvm/test/tools/llvm-readtapi/command-line.test

```diff
@@ -4,12 +4,17 @@
 ; RUN: not llvm-readtapi -merge -compare -compact %t/tmp.tbd %t/tmp2.tbd 2>&1 | FileCheck %s --check-prefix MULTI_ACTION
 // Check unsupported file output format.
 ; RUN: not llvm-readtapi -merge -compact %t/tmp.tbd %t/tmp2.tbd --filetype=tbd-v2 2>&1 | FileCheck %s --check-prefix FILE_FORMAT 
+// Check version printing.
+; RUN: llvm-readtapi -v 2>&1 | FileCheck %s --check-prefix VERSION
+; RUN: llvm-readtapi --version 2>&1 | FileCheck %s --check-prefix VERSION
 
-CHECK: OVERVIEW: LLVM TAPI file reader and manipulator
-CHECK: USAGE: llvm-readtapi [options] [-arch <arch>]* <inputs> [-o <output>]*
-CHECK: OPTIONS:
-CHECK:   -help    display this help
+; CHECK: OVERVIEW: LLVM TAPI file reader and manipulator
```

Comment:
bikeshedding: I am looking for a better word than `manipulator`. Something more along the lines of `modifier`.

Reply:
`transformer`?